.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Setup Go
+      id: setup-go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.14
```

Comment:
Remind me what that means? (I think that we should probably build with 1.15 unless there are known issues).